docs/libraries/nodejs/examples.md

```diff
@@ -379,52 +420,69 @@ run()
 Example of a message with `SignedTransaction` payload:
 ```json
 {
-    'message_id': 'f51fb2839e0a24d5b4a97f1f5721fdac0f1eeafd77645968927f7c2f4b46565b',
-    'network_id': 7712883261355838377,
-    'parents': [
-        '4a84bf1d345a441cfdefd0e71d6efe820c1077e5dda9122a09cbf026132d208c',
-        '6e9153884fd1983be4c27c3ccdc69760b4775484eea498ec0707c2ff8901995e',
-        '7ac1407c88007a54d603400b558d5110f2bbf93a68100fb34f0b40cece9d0868',
-        '9ac0fd457998a1b3ddab9c0014f41344475358ad36c64a4b763de3b51f47c09a'
-    ],
-    'payload': {
-        'transaction': [
-            {
-                'essence': {
-                    'inputs': [
-                        {'transaction_id': '4a34274992474d91cf45366425ad1d4df6042cba64f3b6c07d297a2e6b7154a9', 'index': 0}
-                    ],
-                    'outputs': [
-                        {'address': 'atoi1qp9427varyc05py79ajku89xarfgkj74tpel5egr9y7xu3wpfc4lkpx0l86', 'amount': 10000000},
-                        {'address': 'atoi1qzdnav0zdgd4grn25cnwcuudtahvlhgh0r349ur749y9l03vadrfurhkxwj', 'amount': 100016136757200}
-                    ],
-                    'payload': {
-                        'transaction': None,
-                        'milestone': None,
-                        'indexation': [
-                            {'index': '54414e474c454b495420464155434554', 'data': []}
-                        ],
-                        'receipt': None,
-                        'treasury_transaction': None
-                    }
-                },
-                'unlock_blocks': [
-                    {
-                        'signature': {
-                            'public_key': [...],
-                            'signature': [...]
-                        },
-                        'reference': None
-                    }
-                ]
-            }
-        ],
-        'milestone': None,
-        'indexation': None,
-        'receipt': None,
-        'treasury_transaction': None
-    },
-    'nonce': 1146102
+   "message_id":"f51fb2839e0a24d5b4a97f1f5721fdac0f1eeafd77645968927f7c2f4b46565b",
+   "network_id":7712883261355838377,
+   "parents":[
+      "4a84bf1d345a441cfdefd0e71d6efe820c1077e5dda9122a09cbf026132d208c",
+      "6e9153884fd1983be4c27c3ccdc69760b4775484eea498ec0707c2ff8901995e",
+      "7ac1407c88007a54d603400b558d5110f2bbf93a68100fb34f0b40cece9d0868",
+      "9ac0fd457998a1b3ddab9c0014f41344475358ad36c64a4b763de3b51f47c09a"
+   ],
+   "payload":{
+      "transaction":[
+         {
+            "essence":{
+               "inputs":[
+                  {
+                     "transaction_id":"4a34274992474d91cf45366425ad1d4df6042cba64f3b6c07d297a2e6b7154a9",
+                     "index":0
+                  }
+               ],
+               "outputs":[
+                  {
+                     "address":"atoi1qp9427varyc05py79ajku89xarfgkj74tpel5egr9y7xu3wpfc4lkpx0l86",
```

Comment:
I think here the address is hex encoded because we don't do a manual conversion as for the Python binding